mantis-control-plane/mantis-control-plane-core/src/main/java/io/mantisrx/server/core/utils/ConfigUtils.java

```diff
@@ -64,6 +74,35 @@ public static Method getMethodFromName(String name, Class<? extends CoreConfigur
         }
     }
 
+    /**
+     * This function tries to find a properties file and throws a runtime exception if it cannot.
+     * If found any environment variables matching the property key converted to environment variable
+     * standard form of capital letters and underscores will take precedence.
+     *
+     * @param propFile a property file to find
+     * @return Properties with overrides from the environment variables
+     * @throws RuntimeException if the file can't be found
+     */
+    public static Properties loadProperties(String propFile) {
```

Comment:
I moved this out of `MasterMain` so I could reuse it in the dynamodb extension work.

When config is refactor, we may want to document precedence (props file, java system, system env)?